### PR TITLE
document.execute(async)_script added

### LINKED ISSUE #18: Add methods for executing Javascript in browser
* [x] Add method for synchronous Javascript execution:
```
driver.document.execute_script("return arguments[0] + arguments[1];", 1, 2)
```
* [x] Add method for asynchronous Javascript execution: 
```
driver.document.execute_async_script(
                    "var x = arguments[0]; var y = arguments[1]; var result = x + y; var callback = arguments[arguments.length - 1]; window.setTimeout(callback(result), 100);",
                    40, 2
)
```
* [x] Convert `document.page_source()` to  @property `document.page_source`

### PR DESCRIPTION
**Pull request by pychik**
fixes #18

### CODE REVIEW

File: tests/functional/src/test_document.py [hunk 1]
```diff
@@ -1,5 +1,25 @@
 class TestDocument:
-    def test_document(self, firefox_client):
+    def test_ps(self, firefox_client):
         response = firefox_client.navigation.navigate('https://pychik.github.io/lightning_py/pages/1')\
-                                            .document.get_page_source()
+                                            .document.get_page_source
         assert response.startswith("<html")
+
+    def test_es(self, firefox_client):
```
Comment (vania-pooh) [inline tests/functional/src/test_document.py]: Let's rename to `test_execute_script`.

File: tests/functional/src/test_document.py [hunk 2]
```diff
@@ -1,5 +1,25 @@
 class TestDocument:
-    def test_document(self, firefox_client):
+    def test_ps(self, firefox_client):
```
Comment (vania-pooh) [inline tests/functional/src/test_document.py]: Rename to `test_page_source`.

File: lightning/document.py [hunk 3]
```diff
@@ -10,5 +12,17 @@ def __init__(self, wd):
         super().__init__(wd)
         self._document_instance = DocumentApi(self._api_client)
 
+    @property
     def get_page_source(self) -> str:
```
Comment (vania-pooh) [inline lightning/document.py]: Should be just `page_source`. Property never contains `get_`.

File: tests/functional/src/test_document.py [hunk 4]
```diff
@@ -1,5 +1,25 @@
...
+                                            .document.get_page_source
         assert response.startswith("<html")
+
+    def test_es(self, firefox_client):
+        res = firefox_client.document.execute_script("return arguments[0] + arguments[1];", 1, 2)
+        assert res == 3
+        res = firefox_client.document.execute_script("return arguments[0] + arguments[1];", "1", "2")
+        assert res == "12"
+
+    def test_eas(self, firefox_client):
```
Comment (vania-pooh) [inline tests/functional/src/test_document.py]: Rename to `execute_async_script`.

**Review by vania-pooh — CHANGES_REQUESTED** [review]

File: tests/functional/src/test_document.py (hunk 4 above)
Comment (pychik) [reply tests/functional/src/test_document.py]: solved in b3eabe6a0fda5fc9fd50d8fa8da77df258cc7342

**Review by vania-pooh — APPROVED** [review]

**Pull request merged**